### PR TITLE
allow to generate client only

### PR DESCRIPTION
**Pull request by gitu**
- allows us to only generate clients
- changed the go:generates to use the currently checked out version of the oapi-codegen

### CODE REVIEW

**Comment by gitu** [comment]
@deepmap-marcinr I'm not really happy with the naming of the template & function... do you have a better idea?

File: cmd/oapi-codegen/oapi-codegen.go [hunk 1]
```diff
@@ -64,13 +64,13 @@ func main() {
 
 	swagger, err := util.LoadSwagger(flag.Arg(0))
 	if err != nil {
-		fmt.Fprintf(os.Stderr, "Error loading swagger spec\n: %s", err)
+		_, _ = fmt.Fprintf(os.Stderr, "error loading swagger spec\n: %s", err)
```
Comment (deepmap-marcinr) [inline cmd/oapi-codegen/oapi-codegen.go]: Can we not do _, _ = fmt.Fprintf. I realize lots of IDE's give warnings on uncosumed function output, but it's also pretty clear we're ignoring the result when we leave off the placeholder arguments

File: examples/petstore-expanded/api/doc.go [hunk 2]
```diff
@@ -18,4 +18,4 @@ package api
 
 // Run oapi-codegen to regenerate the petstore server whenever someone does a
 // go generate
-//go:generate sh -c "oapi-codegen --package=api petstore-expanded.yaml > petstore.gen.go"
+//go:generate sh -c "go run github.com/deepmap/oapi-codegen/cmd/oapi-codegen --package=api petstore-expanded.yaml > petstore.gen.go"
```
Comment (deepmap-marcinr) [inline examples/petstore-expanded/api/doc.go]: Cool, thank you. The client generator you ran is out of date, though. Could you rebase and run go generate again to regenerate everything after your change?

File: pkg/codegen/codegen.go [hunk 3]
```diff
@@ -62,7 +62,7 @@ func Generate(swagger *openapi3.Swagger, packageName string, opts Options) (stri
 		return "", fmt.Errorf("error generating Go types for component responses: %s", err)
 	}
 
-	bodiesOut, err := GenerateTypesForRquestBodies(t, swagger.Components.RequestBodies)
```
Comment (deepmap-marcinr) [inline pkg/codegen/codegen.go]: Oh jeez, I can't type. Thank you.

**Comment by deepmap-marcinr** [comment]
I merged your changes myself by pulling your repo into mine. Thank you!

**Pull request closed without merging**